wp-saml-auth.php

```diff
@@ -109,9 +109,62 @@ function wpsa_filter_option( $value, $option_name ) {
 add_filter( 'wp_saml_auth_option', 'wpsa_filter_option', 0, 2 );
 
 /**
+ * Generate a string representation of a function to be used for configuring the plugin.
+ *
+ * @param array
+ * @return string
+ */
+function wpsa_scaffold_config_function( $assoc_args ) {
```

Comment:
If this function is only to be called from `WP_SAML_Auth_CLI` then let's make it a method on that class rather than add to plugin file loaded on all requests.